### ISSUE TITLE
Как использовать альтернативный base url?

### ISSUE THREAD

**Issue by viktor02** [HemulGM/ChatGPT#27]
Я хотел подключить вместо openai [llama-server](https://github.com/abetlen/llama-cpp-python/tree/main#web-server), он эмулирует апи от openai, используя модель LLaMa. Однако когда я ввожу в поле `base url` `http://127.0.0.1:8000`, то это значение не сохраняется, и в чате запрос все равно идет к openai. Что я делаю не так?
![изображение](https://github.com/HemulGM/ChatGPT/assets/19672960/095ea72d-9869-4120-a3b6-b87956dcd91e)

**Comment by HemulGM** [comment]
Сейчас проверю

**Comment by HemulGM** [comment]
Да действительно, я пропустил эту настройку. Спасибо. Сейчас соберу свежий билд под винду

**Comment by HemulGM** [comment]
@viktor02 готово. Можно проверять

**Comment by viktor02** [comment]
Да, спасибо, работает! Хотелось бы еще чтобы работала подгрузка списка списка моделей через /v1/models, но это так, не критично :)

**Comment by HemulGM** [comment]
> Да, спасибо, работает! Хотелось бы еще чтобы работала подгрузка списка списка моделей через /v1/models, но это так, не критично :)

Они там в кучу все. А с chat/completions работают только те что в списке есть. Для других моделей нужно немного менять программу и определять по модели, какой для неё роут.

**Comment by viktor02** [comment]
> > Да, спасибо, работает! Хотелось бы еще чтобы работала подгрузка списка списка моделей через /v1/models, но это так, не критично :)
> 
> Они там в кучу все. А с chat/completions работают только те что в списке есть. Для других моделей нужно немного менять программу и определять по модели, какой для неё роут.

Понял, спасибо
